terraform/testcases/statsd_mock/parameters.tfvars

```diff
@@ -0,0 +1,5 @@
+# this file is defined in validator/src/main/resources/validations
+validation_config="statsd-mocked-metric-validation.yml"
+
+# sample application image to emit the trace data
+sample_app="statsd"
```

Comment:
you might want to add soaking_data_mode and soaking_data_type for performance test on mock test case as well

Reply:
Sure, thanks